markdown/developers/modules.md

```diff
@@ -566,27 +598,67 @@ process {
 
 3. Keywords MUST be all lower case
 
-4. Input and Output sections of the `meta.yaml` SHOULD only have entries of input and output channels
+4. The tools section MUST list every tool used in the module. For example
+
+    ```yml
+    tools:
+      - bowtie2:
+          description:  <...
+          homepage:  <...>
+          documentation:  <...>
+          doi: <...>
+          licence: <...>
+          args_id: <...>
```

Comment:
seems like a good idea to me. Don't find this particularly controversial so far :D

Reply:
For the docs can we highlight what you mean with each section, 4. and 5. are almost identical, I have a hard time spotting the difference